test/mocks.ts

```diff
@@ -8,7 +8,7 @@
 
 import crypto from 'crypto'
 
-const mixAPIServer = 'mix-api.example.com'
+export const mixAPIServer = 'mix-api.example.com'
```

Comment:
Now that I am looking at this, I see  an opportunity to avoid some repetition in our unit test files.

Let's have:
```
const mixAPIServer = 'mix-api.example.com'

export const mixAPIServerURL = `https://${mixAPIServer}`
```

and then in each unit test file, we will replace:

```
import {mixAPIServer} from '../../mocks'
```

with:

```
import {mixAPIServerURL} from '../../mocks'
```

and then replace `serverURL` with `mixAPIServerURL`.